src/components/molecules/UiHeader/UiHeader.vue

```diff
@@ -130,7 +127,6 @@ export interface HeaderProps {
    */
   navigationAttrs?: NavigationAttrsProps;
 }
-export type HeaderAttrsProps = DefineAttrsProps<HeaderProps>;
```

Comment:
IMO it should be there. Check other components like UiBackdrop. Every component export type `<ComponentName>AttrsProps`. Check the doc's https://main--639728541e2039caadfe5404.chromatic.com/?path=/docs/contributing-guide-typescript-guidelines--docs#attrs-props. @AlanLes

Reply:
Even thought it is not used anywhere? In case of `UiBackdrop`, the `BackdropAttrsProps` is used in several spots, but in this particular case - the `HeaderAttrsProps` is not used anywhere.

Reply:
IMO yes. We could add something to HeaderAttrsProps.